core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -1037,6 +1068,34 @@ private void sendNotificationBroadcast(int type, int code) {
         sendBroadcast(intent);
     }
 
+    private void skipEndingPref() {
+        Playable playable = mediaPlayer.getPlayable();
+        if (! (playable instanceof FeedMedia)) {
+            return;
+        }
+
+        int duration = getDuration();
+        int remainingTime = duration - getCurrentPosition();
+
+        FeedMedia feedMedia = (FeedMedia) playable;
+        FeedPreferences preferences = feedMedia.getItem().getFeed().getPreferences();
+        int skipEnd = preferences.getFeedSkipEnding();
+        if (skipEnd > 0 && remainingTime < skipEnd * 1000) {
+            Log.d(TAG, "skipEndingPref: Skipping the remaining " + remainingTime / 1000 + " " + skipEnd );
+            Context context = getApplicationContext();
+            String skipMesg = context.getString(R.string.pref_feed_skip_ending_toast,
+                    skipEnd,
+                    context.getString(R.string.time_seconds));
```

Comment:
I think it would be easier for the translators if you just added the text "Seconds" to `strings.xml`. I do not think that we want to skip other units than seconds. Additionally, this gives a weird mix of languages if "Seconds" is translated but "Skipped first" is not translated.